draft-ietf-quic-manageability.md

```diff
@@ -675,12 +675,12 @@ in the downstream direction, and vice versa.
 
 # Specific Network Management Tasks
 
-In this section, we address specific network management and measurement
+In this section, we review specific network management and measurement
 techniques and how QUIC's design impacts them.
 
 ## Stateful treatment of QUIC traffic {#sec-stateful}
 
-Stateful treatment of QUIC traffic is possible through QUIC traffic and version
+Stateful treatment of QUIC traffic (e.g., at a firewall or NAT middlebox) is possible through QUIC traffic and version
```

Comment:
rewrap this para (and possibly a few others); the 80col linter will choke on it.